.github/workflows/tester.yml

```diff
@@ -0,0 +1,16 @@
+name: unit-tests
```

Comment:
Could you add caching on the `env` folder so we don't need to recreated the environment and install pytest each time we run the tests ?